components/webvr_traits/lib.rs

```diff
@@ -30,3 +30,9 @@ pub use rust_webvr_api::VRLayer as WebVRLayer;
 pub use rust_webvr_api::VRMainThreadHeartbeat as WebVRMainThreadHeartbeat;
 pub use rust_webvr_api::VRPose as WebVRPose;
 pub use rust_webvr_api::VRStageParameters as WebVRStageParameters;
+
+#[derive(Deserialize, Serialize)]
+pub struct WebVRPoseInformation {
```

Comment:
bikeshed: not sure `PoseInformation` is the best name for data about the frame + gamepad state.

Reply:
Got a suggestion for a replacement?

Reply:
`WebVRDeviceState`?